source/connect-to-govwifi/device-blackberry.html.erb

```diff
@@ -43,14 +42,9 @@ description: Follow these instructions to connect your Blackberry to GovWifi
           </li>
         </ol>
         <p class="govuk-body">You should now be connected to GovWifi.</p>
-        <p class="govuk-body"><strong>Support</strong></p>
-        <p class="govuk-body">You may get a message that says 'Wi-Fi Connection Status, Attempt 1 of 3, Connecting to network' and another that says ‘Acquiring IP address'.  You don't need to do anything.</p>
 
-        <%= image_tag "troubleshooting/blackberry-step-7-a.jpg", class: "img-fluid", alt: "Screenshot of WiFi connection status on Blackberry" %>
-        <%= image_tag "troubleshooting/blackberry-step-7-b.jpg", class: "img-fluid", alt: "Screenshot of WiFi connection update on Blackberry" %>
-
-        <h2 class="govuk-heading-m">BlackBerry Android</h2>
-        <p class="govuk-body">If your BlackBerry device uses Android, follow the <%= link_to "Android instructions", "/connect-to-govwifi/device-android", class: "govuk-link" %>.</p>
+        <h2 class="govuk-heading-m">Support</h2>
```

Comment:
this appears to remove a few images from the page - are the images still needed? Because they're not removed from the repo in this PR

Reply:
It removes a link that I think is helpful

I'd use both new and old text like below:

If your device was made before October 2016, you're probably running the BlackBerry operating system (OS). Follow these instructions to connect to GovWifi.

> If your device uses the Android operating system or it was made after October 2016, follow the follow the <%= link_to "Android instructions", "/connect-to-govwifi/device-android", class: "govuk-link" %>.

Reply:
Oh, no I don't think they are - this whole 'support' section seemed pointless as there's no user action at all. Didn't realise I had to remove them elsewhere too! Thanks

Reply:
Ian - I think they do essentially the same thing? All we need to do is point people running Android elsewhere and I'd rather make this as short as possible. And (new) line 17 already says 'follow the steps below' so I don't think we need to duplicate that instruction here too.